src/components/ContentWrapper.jsx

```diff
@@ -40,20 +41,37 @@ const ContentWrapper = (props) => {
   const [isAuth, setIsAuth] = useState(false);
   const [collapsed, setCollapsed] = useState(false);
 
-  const { experimentId, experimentData, children } = props;
+  const { routeExperimentId, experimentData, children } = props;
```

Comment:
Changed this to differentiate between the experimentId in the URL and the one unders activeproject.experiments[0]

Reply:
When does this happen? The users can't change experiment if they are not in `data-management` right? and Data Management does not have a route in its URL